### PR TITLE
#168052207 Create and verify Json Web Token

### PR DESCRIPTION
**Pull request by dieudonneAwa**
**What does this PR do?**
- Create json web token for user registration

**Description of Task to be completed?**
- Create a file to process tokens
- This file exports a class called ProcessToken that has two helper methods i.e `createToken` and `verifyToken`
- Import this file and destructure out the helper method you want to use
- `createToken` will create token by passing a user object as the payload.
- 'verifyToken` will verify the token 

**How should this be manually tested?**
After cloning the repo,
- run `npm i` to install `jsonwebtoken` dependency.
- import the file whenever you want to use and and destructure the class methods from it
- pass in the user object from which you want to create the token
- possibly store the returned value in a variable called `token`
- you can now use the token 

**Any background context you want to provide?**
- [JSON WEB TOKEN](https://www.npmjs.com/package/jsonwebtoken)

 **What are the relevant stories?**
[#168052207](https://www.pivotaltracker.com/n/projects/2381984/stories/168052207)

**Screenshots (if appropriate)**
- None

**Questions:**
- None

### CODE REVIEW

**Review by justinefe — APPROVED** [review]

**Review by Abobos — APPROVED** [review]

**Review by Abobos — APPROVED** [review]

File: tests/tokenProcessor.test.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  describe('Test Create Token', () => {
+    it('Should return a token string', (done) => {
+      expect(token).to.be.a('string');
+      done();
+    });
+  });
+
+  describe('Test Verify Token', () => {
+    it('Should return an object', (done) => {
+      expect(verifiedToken).to.be.an('object');
```
Comment (chukwuemekachm) [inline tests/tokenProcessor.test.js:23]: This should decode the token you created on line 12 and assert that it contains an email property with the value on line 11.

**Review by chukwuemekachm — CHANGES_REQUESTED** [review]
Nice work @dieudonneAwa.
just a single concern.

File: tests/tokenProcessor.test.js (hunk 1 above)
Comment (dieudonneAwa) [reply tests/tokenProcessor.test.js:23]: okay

**Review by chukwuemekachm — APPROVED** [review]
LGTM

**Pull request merged**